index.html

```diff
@@ -543,7 +543,8 @@
 
 ### Glossy-diffuse
 
-The glossy-diffuse slab represents a dielectric with rough GGX [microfacet](index.html#model/microfacetmodel) surface BSDF (whose "specular" parameters are described in the Dielectric base section), embedding a semi-infinite bulk of extremely dense scattering material.
+The glossy-diffuse slab represents a dielectric with rough GGX [microfacet](index.html#model/microfacetmodel) surface BSDF (whose "specular" parameters are described in the Dielectric base section), embedding a semi-infinite bulk of extremely dense scattering material. The resulting BRDF of the glossy-diffuse slab is the combination of a "glossy" specular lobe provided by reflection from the dielectric interface, and a diffuse lobe provided by bounces off the substrate. This models for example the reflection from shiny, totally opaque surfaces such as dense plastic, rock, and concrete.
```

Comment:
Minor remark: the Dielectric-base section already describes the dielectric surface, so mabe we don't need to repeat that here, especially since we don't in the Subsurface and Translucent base sub-sections.

The reason I noticed this was because I first understood this paragraph as saying both the dielectric and the diffuse had a GGX NDF, and it took me a couple more reading to understand the importance of "embedding" in this description.

Reply:
We do mention it also in the Subsurface and Translucent base sections:
>As in the cases of the glossy-diffuse slab and the translucent-base, the subsurface is bounded by a dielectric interface with BSDF fdielectric which generates the primary specular reflection lobe parametrized via the “specular” parameters as described in the Dielectric base section. Combined with this is the reflection generated by light which is transmitted through the dielectric interface into the underlying embedded subsurface medium, where it scatters around and eventually transmits back out.

>As for the dielectric slabs defined in the Glossy-diffuse and Subsurface section, the top interface is described by a rough GGX microfacet BSDF fdielectric whose “specular” parameters are described in the Dielectric base section.

So for consistency I think we should mention it as well here.
I agree the wording could be better though, to make it clear that the "dielectric with rough GGX microfacet surface" being referred to is the same one as in the other sections (and we should link to the Dielectric base section).

Reply:
To be fair, we're in nitpicking territory here, and I'm perfectly fine with the PR being merged as it is.

Reply:
Thanks for clarifying it.